src/jit/codegen.h

```diff
@@ -935,6 +935,7 @@ XXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXX
     void genHWIntrinsic_R_R_RM_R(GenTreeHWIntrinsic* node, instruction ins);
     void genHWIntrinsic_R_R_R_RM(
         instruction ins, emitAttr attr, regNumber targetReg, regNumber op1Reg, regNumber op2Reg, GenTree* op3);
+    void genBASEIntrinsic(GenTreeHWIntrinsic* node);
```

Comment:
`BASE` isn't an acronym, so this should be `Base`, I think.

Reply:
Fixed.